src/v/kafka/server/handlers/alter_partition_reassignments.cc

```diff
@@ -200,6 +200,18 @@ ss::future<response_ptr> alter_partition_reassignments_handler::handle(
     log_request(ctx.header(), request);
     alter_partition_reassignments_response resp;
 
+    if (!config::shard_local_cfg().kafka_enable_partition_reassignment()) {
```

Comment:
Should this also go in `list_partition_reassignments.cc` ?

Reply:
Hmm, good question. From the perspective of protecting against what we care about, it doesn't seem to matter. I don't know if disabling the list interface would provide a better experience.

Reply:
Well, the list interface will (or should) return and empty response when the `kafka_enable_partition_reassignment=False` because there should be 0 reassignments.
We should probably include that in the test then.

Reply:
There could be reassignments via the AdminAPI or the auto-balancer

Reply:
>There could be reassignments via the AdminAPI or the auto-balancer

yeh it might be useful if some tool was monitoring rebalances but didn't actually want to change things.